src/js/components/receiving/modals/EditLineModal.jsx

```diff
@@ -85,6 +86,7 @@ const FIELDS = {
 function validate(values) {
   const errors = {};
   errors.lines = [];
+  const date = moment('01/01/2000', 'MM/DD/YYYY');
```

Comment:
Again we need to pull this in via the config properties. If possible, add it to whatever we use to load the data for this page/modal. Or we need a more generic app config API.